recipes/react-router/package.json

```diff
@@ -0,0 +1,34 @@
+{
+  "name": "react-router-recipe",
+  "private": true,
+  "type": "module",
+  "version": "1.0.0",
+  "scripts": {
+    "build": "cross-env NODE_ENV=production react-router build",
+    "dev": "react-router dev",
+    "start": "cross-env NODE_ENV=production react-router-serve ./build/server/index.js",
+    "typecheck": "react-router typegen && tsc"
+  },
+  "dependencies": {
+    "@measured/puck": "^0.17.1",
+    "@react-router/node": "^7.0.2",
+    "@react-router/serve": "^7.0.2",
+    "isbot": "^5.1.17",
```

Comment:
Noticed this is in the standard react-router template, but is this dep necessary?

Reply:
Defaults from the base RR template, it can be removed but we'd need to add a new server entry file because RR expects the dependency to be there.